### ISSUE TITLE
Clarification on LiDAR Domain Generalisation and Fine-Tuning Strategy

### ISSUE THREAD

**Issue by SeungGyu-Song** [minwoo0611/ImLPR#3]
Hi Minwoo,

First of all, thanks for your great work Minwoo.

Your research sparked my interest in generalisation tasks with pre-trained foundation models.
I couldn't thank enough for that. 

I have a question regarding the adaptation to the LiDAR domain.

In my view, the proposed model does not include moduels designed for LiDAR-specific properties, but it was interestingly able to be adapted to the entirely new domain.

The side adapters look similar to ones from SelaVPR++ which is for VPR (easily noticeable from its name) and they just seem to extract multi-dimensional features due to the different CNN kernel sizes. 

My question is how your model could successfully be adapted to the new sensor domain with the MCA for fine-tuning.
If one of your novel loss function, InfoNCE loss, was the key for fine-tuning and adaptation, I wonder why you chose MCA for fine-tuning the foundation model.
Although RIV images embrace the specific properties, I am curious how the frozen backbone model was able to extract meaningful features learned from RGB images. 

Additionally, I would be grateful if you could share with me about any experiences to be potentially helpful to those who could extend your work:)

Best regards,
Seunggyu Song
